### DISCUSSION TITLE
Limit maximum resolution asked to play

### DISCUSSION THREAD

**Discussion by sciamanna** [croneter/PlexKodiConnect#1390] [Q&A] [answered]
Is it possible to get PKC to only ask Plex for 1080p versions when both 1080p and 4K versions of the same media item are available? I want a remote PKC installation to not try asking for 4K versions of files while still allowing a local PKC client to play 4K versions in the same library (I don't want to use multiple Plex libraries as they are a pain to maintain and take up a lot of space for the metadata).

I was hoping _max_quality in playback_decision.py might be the answer

def _max_quality():
    return {
        'maxVideoBitrate': MAX_SIGNED_INT,
        'videoResolution': '3840x2160',  # 4K
        'videoQuality': 100,
        'mediaBufferSize': int(float(utils.settings('kodi_video_cache')) / 1024.0),
    }

Would changing videoResolution to a user definable setting help in this use case? If that was set to 1920x1080 would that persuade Plex to use the non 4K version?

Thanks!

**Comment by croneter** [comment, marked as answer]
No that's not yet possible. Kindly open a feature request: http://feathub.com/croneter/PlexKodiConnect
